packages/datadog-plugin-aws-sdk/src/base.js

```diff
@@ -163,11 +163,34 @@ function normalizeConfig (config, serviceIdentifier) {
       break
   }
 
-  return Object.assign({}, config, specificConfig, {
+  const baseAWSBatchPropagationValue = process.env.DD_TRACE_AWS_SDK_BATCH_PROPAGATION_ENABLED
```

Comment:
I think these should have correlating config values for when a user calls `tracer.init({})` manually.

Reply:
I simplified this, they are now one and the same.